From: Michael Olbrich <m.olbrich@pengutronix.de>
Subject: remove wrong include path.

Don't use $(prefix)/include as include path. It can break things
when cross-compiling with DESTDIR and prefix=/usr

# 20091215 mol: this patch is already upstream: 8337651f91372ec6f16c1bdc7fe2c9af2d450db8

Signed-off-by: Michael Olbrich <m.olbrich@pengutronix.de>

---
 configure.ac |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: xf86-video-apm-1.2.2/configure.ac
===================================================================
--- xf86-video-apm-1.2.2.orig/configure.ac
+++ xf86-video-apm-1.2.2/configure.ac
@@ -67,7 +67,7 @@ AC_SUBST([XF86RUSH_LIBS])
 AC_SUBST([XF86RUSH_CFLAGS])
 
 CFLAGS="$CFLAGS $XORG_CFLAGS $XF86RUSH_CFLAGS"' -I$(top_srcdir)/src'
-INCLUDES="$XORG_INCS -I${sdkdir} $XF86RUSH_INCS "'-I$(top_srcdir)/src -I$(prefix)/include'
+INCLUDES="$XORG_INCS -I${sdkdir} $XF86RUSH_INCS "'-I$(top_srcdir)/src'
 AC_SUBST([CFLAGS])
 AC_SUBST([INCLUDES])
 
